src/main/scala/com/fulcrumgenomics/fastq/FastqSource.scala

```diff
@@ -59,14 +59,14 @@ object FastqSource {
     * @param sources a Seq of one or more FastqSource objects
     * @return an Iterator that returns a Seq of FastqRecord, one per source
     */
-  def zipped(sources: Seq[FastqSource]): Iterator[Seq[FastqRecord]] = new Iterator[Seq[FastqRecord]] {
```

Comment:
I'm pretty sure I use this method elsewhere.  Can you keep both please and have vector this one to the one that takes iterators?